Content.Server/CartridgeLoader/Cartridges/NotekeeperCartridgeSystem.cs

```diff
@@ -36,16 +39,19 @@ private void OnUiMessage(EntityUid uid, NotekeeperCartridgeComponent component,
         if (message.Action == NotekeeperUiAction.Add)
         {
             component.Notes.Add(message.Note);
+            _adminLogger.Add(LogType.AdminMessage, LogImpact.Low,
```

Comment:
AdminMessage is the wrong LogType for this, but not sure which one to use instead. Let me ping an admin.
@nikthechampiongr 

Otherwise the code looks good.

Reply:
I can't find a log type that currently exists that would be appropriate. You will probably need to make a new type. AdminMessage tho is definitely a miscategorisation tho.